packages/host/app/components/operator-mode/delete-modal.gts

```diff
@@ -0,0 +1,148 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { enqueueTask, dropTask, timeout, all } from 'ember-concurrency';
+import { Deferred } from '@cardstack/runtime-common';
+import { on } from '@ember/modifier';
+import { fn } from '@ember/helper';
+import { action } from '@ember/object';
+import { Modal, BoxelButton } from '@cardstack/boxel-ui';
+import cssVar from '@cardstack/boxel-ui/helpers/css-var';
+import type { Card } from 'https://cardstack.com/base/card-api';
+
+interface Signature {
+  Args: {
+    onCreate: (instance: DeleteModal) => void;
+  };
+}
+
+export default class DeleteModal extends Component<Signature> {
+  <template>
+    <Modal
+      data-test-delete-modal={{this.currentConfirmation.card.id}}
+      @layer='urgent'
+      @size='x-small'
+      @isOpen={{this.showModal}}
+      @onClose={{fn this.choose false}}
+      style={{cssVar boxel-modal-offset-top='40vh'}}
+    >
+      <div class='delete'>
+        <div class='content'>Delete the card</div>
+        <div class='content'><strong
+          >{{this.currentConfirmation.card.title}}</strong>?</div>
```

Comment:
Is this Prettier? whyeee 😭

Reply:
It might be because the tags are connected without white space. I’ll add some white space so that our eyes will stop bleeding.